src/test/ui/consts/const-eval/transmute-const.rs

```diff
@@ -0,0 +1,18 @@
+// Copyright 2018 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+#![feature(const_transmute)]
+
+use std::mem;
```

Comment:
can you also add a test that checks that `static FOO: bool = unsafe { mem::transmute(3) };` is an error?

Reply:
The tests exists now but it fails saying the sizes are wrong.

Likely you meant `mem::transmute(3u8)`?

Reply:
I'm slightly confused, because I built it on my box and it passes, as does the travis build.